src/Facebook/InstantArticles/AMP/AMPArticle.php

```diff
@@ -208,7 +208,7 @@ public function buildAnalytics($context = null)
         if ($context === null) {
             $context = $this->getContext();
```

Comment:
When trying to add a failing test, I realized that the method `getContext()` isn't defined in this class, so the test fails because it doesn't exist (rather than because the `Exception` class doesn't exist in the current namespace.

It appears that it should return an instance of [`AMPContext`](https://github.com/facebook/facebook-instant-articles-sdk-extensions-in-php/blob/682568807bfcf32e7c28a57ce818668a0aa9fdd9/src/Facebook/InstantArticles/AMP/AMPContext.php), maybe you can suggest how it should be created?